android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -537,6 +542,22 @@ private void collectDeviceIdentifiers(final Result result) {
         result.success(null);
     }
 
+    private void canMakePayments(List<String> features, final Result result) {
+        CommonKt.canMakePayments(applicationContext,
+                features,
+                new OnResultAny<Boolean>() {
+                    @Override
+                    public void onReceived(Boolean received) {
+                        result.success(received);
+                    }
+
+                    @Override
+                    public void onError(@org.jetbrains.annotations.Nullable ErrorContainer errorContainer) {
```

Comment:
we're already importing `@Nullable` in line 7, right? can we just use `@Nullable` here? I believe we do that in other places